src/components/Incidents/IncidentsPage.jsx

```diff
@@ -141,6 +137,38 @@ const IncidentsPage = ({ customDataSource, namespace }) => {
     })();
   }, [span]);
 
+  React.useEffect(() => {
+    (async () => {
```

Comment:
we might not need async here as the only await used is inside the `timeRanges.map`